iPayments.js

```diff
@@ -0,0 +1,145 @@
+let libPrefix = "iPayments" 
+var price_api_url = "https://api.i-payments.site/check/price" 
```

Comment:
var api_url = "https://api.i-payments.site/"

use this only.
Then you can use later:
api_url + "/check/price" and etc

Reply:
Using this i got 0 error